kb/2023-02-01/scan_and_repair_filesystem.md

```diff
@@ -65,11 +65,31 @@ This section will introduce how to scan the filesystem (e.g., XFS, EXT4) using r
 
 Before scanning, you need to know the filesystem's device/partition.
 
-- Find the filesystem's device by running the `dmesg` command. The most recent device should be what we wanted because we attach the latest.
-- You should now know the filesystem's partition. In the example below, sdd3 is the filesystem's partition.
-
-![finding the related device](./imgs/finding_related_device.png)
-
+- Find the filesystem's device with the following instruction by checking the `Major:Minor` for the specific device.
+
+  1. Try to specify the volume named `pvc-ea7536c0-301f-479e-b2a2-e40ddc864b58`.
+  ```
+  // find the `Major:Minor` for it
+  harvester-node-0:~ # ls /dev/longhorn/pvc-ea7536c0-301f-479e-b2a2-e40ddc864b58 -al
+  brw-rw---- 1 root root 8, 0 Oct 23 14:43 /dev/longhorn/pvc-ea7536c0-301f-479e-b2a2-e40ddc864b58
+  
+  // `Major:Minor` is `8:0` from the above output
+  ```
```

Comment:
1. I moved the comments out of the code block.
2. I could not use ``` to mark the code block because GitHub's suggestion feature also uses it. Please replace the instances of 'BACKTICKS'.